Automattic/woocommerce-payments
# Update/2194 replace connect account page exp

Fixes #2194 

#### Changes proposed in this Pull Request

This PR removes `wcpay_inapp_payments_connect_page_v2` experiment and replaces the default experience with the treatment experience. The treatment experience is now the default.

1. Renamed `connect-account-page-variant` to `connect-account-page`
2. Removed `connect-account-page-experiment` component.

Update 1: I'm not exactly sure why add641f73da3863d17d0b831ea35054e8b7c4f9f and add641f73da3863d17d0b831ea35054e8b7c4f9f show up in the commits tab 🤔 


#### Testing instructions

1. It is best to start testing in an Incognito window if you have been testing this experiment.
2. Navigate to Payments and confirm the treatment experience is shown.

#### Screenshots
![Screen Shot 2021-06-15 at 4 15 20 PM](https://user-images.githubusercontent.com/4723145/122135735-e6ee7180-cdf5-11eb-992d-62eed3602ab4.jpg)
